### PR TITLE
Updated release notes

### PR DESCRIPTION
**Pull request by tst-nfarah**
This is a fix for #1045 

Added 2 fixed bugs and one enhancement.

### CODE REVIEW

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 1]
```diff
@@ -69,6 +70,23 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>881</td>
+            <td>Error Saving Map Layer with Icon</td>
+            <td>
+              The registry of image decoders in the third-party skia library,
+              used in handling icons for map layers, is now being populated.
+              Saving map layers with icons now works.
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: can we reword "now works" to "now functions correctly"?

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 2]
```diff
@@ -69,6 +70,23 @@ <h5>
...
+            </td>
+          </tr>
+          <tr>
+            <td>882</td>
+            <td>GEE Portable Build script fails on CentOS 6</td>
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: lowercase "Build" in "Build script"

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 3]
```diff
@@ -69,6 +70,23 @@ <h5>
...
+            <td>
+              Added code to use g++ from the devtoolset2 installation on 
+              Cent OS 6, if the default g++ compiler version is too low.
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: Should be "CentOS 6" - watch the space. Also, no need for comma.

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 4]
```diff
@@ -79,14 +97,14 @@ <h5>
             </td>
           </tr>
           <tr>
-            <td>881</td>
-            <td>Error Saving Map Layer with Icon</td>
+            <td>946</td>
+            <td>Panoramic Streetview JS error</td>
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: Expand "JS" to "JavaScript", or if the errors only appear in the JS console, the use "debug console"

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 5]
```diff
@@ -79,14 +97,14 @@ <h5>
...
             <td>
-              The registry of image decoders in the third-party skia library,
-              used in handling icons for map layers, is now being populated.
-              Saving map layers with icons now works.
+              The codes used in two .js files were from different versions of the library.
+              Code was changed to be consistent with the other dependinding .js.
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: spelling of "dependinding" - is "depending" the correct word here?
Also, end with "the other depending .js files."

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 6]
```diff
@@ -385,6 +403,16 @@ <h5>
           <td>Geserver fails to startup fully due to conflicting protobuf library</td>
           <td>Run <code>pip uninstall protobuf</code> to uninstall the protobuf library installed by pip. </td>
         </tr>
+        <tr>
+          <td>1012</td>
+          <td>Cutter produces incorrect very low resolution result from 2d mercator map</td>
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: Are we using "2d" or "2D" - let's be consistent.

File: docs/geedocs/5.2.4/answer/7160004.html [hunk 7]
```diff
@@ -385,6 +403,16 @@ <h5>
...
+          <td>No current work around. </td>
```
Comment (tst-jcupp) [inline docs/geedocs/5.2.4/answer/7160004.html]: reword to "No known workaround"

File: docs/geedocs/5.2.4/answer/7160004.html (hunk 1 above)
Comment (tst-nfarah) [reply docs/geedocs/5.2.4/answer/7160004.html]: ok

File: docs/geedocs/5.2.4/answer/7160004.html (hunk 7 above)
Comment (tst-nfarah) [reply docs/geedocs/5.2.4/answer/7160004.html]: We are using in other known issues the wording "No current work around". If we change here we need to changes in other places to be consistent.

Comment (tst-nfarah) [reply docs/geedocs/5.2.4/answer/7160004.html]: Ok. I went through them and changed them all.

**Pull request merged**